tests/foreman/cli/test_contentview.py

```diff
@@ -1731,7 +1725,7 @@ def test_positive_subscribe_chost_by_id_using_puppet_content(self):
             # Associate puppet module to CV
             ContentView.puppet_module_add({
                 u'content-view-id': content_view['id'],
-                u'name': puppet_module['name'],
+                u'uuid': puppet_module['uuid'],
```

Comment:
Changed to `uuid` to have both ways of addition (author + name an uuid only) covered in tests.